receiver/postgresqlreceiver/testdata/scraper/top-query/expected.yaml

```diff
@@ -49,6 +49,9 @@ resourceLogs:
               - key: postgresql.total_plan_time 
                 value: 
                   doubleValue: 1
+              - key: postgresql.query_plan
+                value:
+                  stringValue: "{}"
```

Comment:
Let's update the test to reflect the change?

Reply:
updated